scripts/jenkins-yoctobuild-build.sh

```diff
@@ -188,15 +191,15 @@ EOF
     if [ -n "$mender_on_exact_tag" ]; then
         cat >> $BUILDDIR/conf/local.conf <<EOF
 # MEN-2948: Renamed mender recipe -> mender-client
-# But the `mender` reference has to be kept for backwards compatibility
+# But the "mender" reference has to be kept for backwards compatibility
 # with 2.1.x, 2.2.x, and 2.3.x
 PREFERRED_VERSION_pn-mender = "$mender_on_exact_tag"
 PREFERRED_VERSION_pn-mender-client = "$mender_on_exact_tag"
 EOF
     else
         cat >> $BUILDDIR/conf/local.conf <<EOF
 # MEN-2948: Renamed mender recipe -> mender-client
-# But the `mender` reference has to be kept for backwards compatibility
+# But the "mender" reference has to be kept for backwards compatibility
```

Comment:
Thanks for fixing this! The funny part is that I have a memory of having fixed it already...